modules/lang-painless/src/main/java/org/elasticsearch/painless/PainlessExecuteAction.java

```diff
@@ -347,11 +348,13 @@ public void writeTo(StreamOutput out) throws IOException {
         // For testing only:
         @Override
         public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+            builder.startObject();
```

Comment:
Hi @martijnvg - I am checking if this structural change impacts any of the public-facing REST API endpoints and therefore if the .NET client needs changing. I see this method is marked as `// For testing only` - but I wanted to double check with yourself first.

Thanks!

Reply:
Does not affect client response format, this is used for testing only.